plugins/source/aws/resources/services/ssoadmin/permission_sets_fetch.go

```diff
@@ -10,6 +11,29 @@ import (
 	"github.com/cloudquery/plugin-sdk/schema"
 )
 
+func getSsoadminPermissionSetInlinePolicy(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	svc := meta.(*client.Client).Services().Ssoadmin
+	permissionSetARN := resource.Item.(*types.PermissionSet).PermissionSetArn
+	instanceARN := resource.Parent.Item.(types.InstanceMetadata).InstanceArn
+	config := ssoadmin.GetInlinePolicyForPermissionSetInput{
+		InstanceArn:      instanceARN,
+		PermissionSetArn: permissionSetARN,
+	}
+
+	response, err := svc.GetInlinePolicyForPermissionSet(ctx, &config)
+	if err != nil {
+		return err
+	}
+
+	var inlinePolicy map[string]interface{}
+	err = json.Unmarshal([]byte(*response.InlinePolicy), &inlinePolicy)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
```

You shouldn't need to this `Unmarshal` call... It should be done automatically by the SDK

Reply:
This is the thing I am talking about in the discord thread - I don't see the contents of the policy show up in the table unless I do this

Reply:
@hermanschaaf @bbernays I tried it again and yes it works on my end too. I will merge the suggestion now thank you both.